homeassistant/components/obihai/__init__.py

```diff
@@ -1,13 +1,41 @@
 """The Obihai integration."""
 
+from pyobihai import PyObihai
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
 from homeassistant.core import HomeAssistant
+import homeassistant.helpers.config_validation as cv
+
+from .connectivity import ObihaiConnection
+from .const import DEFAULT_PASSWORD, DEFAULT_USERNAME, DOMAIN, OBIHAI, PLATFORMS
 
-from .const import PLATFORMS
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
This wasn't in the original feedback, but it seems to me the PLATFORM_SCHEMA isn't specific to the sensor entity and so I moved it here.  If this isn't the right place, I can move it elsewhere.

Reply:
This change is wrong. It is only for the deprecated YAML and shouldn't be moved/adjusted

Reply:
Doh!  Thanks, right.

Reply:
You can add a comment to remind yourself :)